doc/attr.xml

```diff
@@ -1352,3 +1352,38 @@ gap> HamiltonianPath(g);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="VertexConnectivity">
+<ManSection>
+  <Attr Name="VertexConnectivity" Arg="digraph"/>
+  <Returns>An non-negative integer.</Returns>
+  <Description>
+    For a digraph <A>digraph</A> with set of vertices <C>V</C>, the attribute
+    <C>VertexConnectivity(<A>digraph</A>)</C> returns the least cardinality
+    <C>|S|</C> of a subset <C>S</C> of <C>V</C> such that their removal either
+    disconnects <A>digraph</A>, makes it trivial, or makes it have no vertices.
```

Comment:
`such that their removal either disconnects <A>digraph</A>, makes it trivial, or makes it have no vertices.`

I think this would be better phrased as

`such that the induced subdigraph of <A>digraph</A> on <C>V\S</C> is disconnected, or has at most one vertex.`